css/style-desktop.css

```diff
@@ -192,8 +197,8 @@
     justify-content: center;
```

Comment:
> This comment refers to line #144

- [ ] [_**OPTIONAL**_] Kindly fix the width set to the recent works container which is causing the whole section to overflow outside the window as shown below: 
![image](https://user-images.githubusercontent.com/61377322/146308659-84301de6-3d70-4ffe-adf7-bb3eb24c37bf.png)